uint32 robotNumber
float32[] positions
float32[] velocities
float32 heading
